agent-any/src/AnyAgent.cc

```diff
@@ -675,7 +675,7 @@ static FILE* program_stream(const char *s, SCRAgent* agent)
 
               // Do not allow touch outside of chroot especially `cd ~` can
               // cause errors
-              chdir("/");
+              if(chdir("/"));
```

Comment:
better to behave same as above so `_exit(1)` if it cannot change dir ( very unlikely ).